src/whir/committer/reader.rs

```diff
@@ -35,6 +47,28 @@ where
         &self,
         verifier_state: &mut VerifierState,
     ) -> ProofResult<ParsedCommitment<F, MerkleConfig::InnerDigest>>
+    where
+        VerifierState: UnitToBytes
+            + FieldToUnitDeserialize<F>
+            + UnitToField<F>
+            + DigestToUnitDeserialize<MerkleConfig>
+            + BytesToUnitDeserialize,
+    {
+        let mut batch_size_le_bytes: [u8; 4] = [0, 0, 0, 0];
```

Comment:
This should not be read from the transcript, it's part of the protocol parameters so we can read it from there.